android_default_browser_metrics.view.lkml

```diff
@@ -0,0 +1,77 @@
+
+view: android_default_browser_metrics {
+  derived_table: {
+    sql: WITH dau_segments AS 
+          (SELECT submission_date, SUM(DAU) as dau
+          FROM `moz-fx-data-shared-prod.telemetry.active_users_aggregates` 
+          WHERE app_name = 'Fenix'
+          --AND channel = 'release'
+          AND submission_date >= '2021-01-01'
+          GROUP BY 1),
+          
+      
+      product_features AS
+      (SELECT
+          client_info.client_id,
+          DATE(submission_timestamp) as submission_date,
+          COALESCE(SUM(CASE WHEN metrics.boolean.metrics_default_browser THEN 1 ELSE 0 END)) AS metrics_default_browser
+          FROM `mozdata.fenix.metrics`
+      where DATE(submission_timestamp) >= '2021-01-01'
+      AND sample_id = 0
```

Comment:
is this sample_id filter intended? I notice you don't have it any other place in the query

Reply:
nevermind, i see you multiply by 100 below.